qa/workunits/cephtool/test.sh

```diff
@@ -197,6 +197,12 @@ function ceph_watch_wait()
     fi
 }
 
+function test_mon_osd_modify_not_exsited_option()
+{
+  ceph daemon osd.0 config set aaaaa 1 | grep "no such an option"
+  ceph daemon mon.a config set aaaaa 1 | grep "no such an option"
```

Comment:
@liupan1111 i can take this PR from you and continue on it if you agree. i think it might not a straightforward change, but not that complicated either.